packages/ripple-nuxt-tide/modules/publication/pages/PrintAll.vue

```diff
@@ -120,6 +133,11 @@ export default {
   },
   async asyncData ({ app, route }) {
     const publication = await app.$tide.getPageByPath('/' + route.params.publicationname)
+    if (publication) {
+      const pubCompMapping = await app.$tideMapping.get(publication.field_landing_page_component, 'landingPageComponents')
+      publication.components = app.$tide.getDynamicComponents(pubCompMapping, true)
```

Comment:
hi @tim-yao , Do you think we should check "pubCompMapping" for not null or not empty as well!